platform.md

```diff
@@ -656,6 +661,7 @@ Usage:
 | `<app>`          | `CNB_APP_DIR`          | `/workspace`             | Path to application directory                                                                  |
 | `<build-config>` | `CNB_BUILD_CONFIG_DIR` | `/cnb/build-config`      | Path to build config directory                                                                 |
 | `<buildpacks>`   | `CNB_BUILDPACKS_DIR`   | `/cnb/buildpacks`        | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout)) |
+| `<exec-env>`     | `CNB_EXEC_ENV`         | `production`             | Target execution environment. Standard values: "production", "test", "development".            |
```

Comment:
We probably want `CNB_EXEC_ENV` in restorer as well. In order to do things like skip cache restore. 

`Extender` also needs it I would think?